.bazelrc

```diff
@@ -0,0 +1 @@
+build --action_env=BAZEL_CXXOPTS=-std=c++17
```

Comment:
If binaryen is used as a dependency this will have no effect.  What is the purpose of BAZEL_CXXOPTS?  If all C++ targets require c++17, that can be added to each target using a global variable, e.g.

```
load("//:CONFIG.bzl", "CXXOPTS")
...
cc_library(
    ...
    copts = CXXOPTS + [ ... ],
   ...
)

Reply:
Suggestion: put the c++17 arg in the variable:

```
BINARYEN_COPTS = [
    "-fexceptions",
    "-std=c++17"
]
```